Presentation.Web/app/components/global-admin/global-admin-local-admins-root.component.ts

```diff
@@ -0,0 +1,158 @@
+            const self = this;
+            this.organizationRightService.create(newOrgId, userToUpdateId, roleId)
+                .then(() => {
+                    msg.toSuccessMessage(userName + " er blevet lokal administrator for " + orgName);
+                    if (self.userId === userToUpdateId) {
+                        // Reload user
+                        self.userService.reAuthorize();
+                    }
+
+                    self.loadData();
+                }, () => {
+                    msg.toErrorMessage("Kunne ikke gøre " + userName + " til lokal administrator for " + orgName);
+                });
+
+            this.newOrg = null;
+            this.newUser = null;
+        }
+
+        deleteLocalAdmin(rightId: number) {
+            const localAdminRow = _.find(this.localAdmins, { id: rightId });
+            if (!localAdminRow)
+                return null;
+
+            const right = localAdminRow.objectContext;
+
+            const organizationId = right.organizationId;
+            const roleId = right.role;
+            var userId = right.userId;
+
+            return this.organizationRightService.remove(this.currentOrganizationId, organizationId, roleId, userId)
+                .then(() => {
+                    if (userId === this.userId) {
+                        this.userService.reAuthorize();
+                    }
+                    this.loadData();
+                });
+        }
+        
+        getOrganizationSelectOptions() {
+            return this.select2LoadingService.loadSelect2WithDataHandler("api/organization", true, ["take=100", "orgId=" + this.currentOrganizationId], (item, items) => {
+                items.push({
+                    id: item.id,
+                    text: item.name ? item.name : 'Unavngiven',
+                    cvr: item.cvrNumber
+                });
+            }, "q", Helpers.Select2OptionsFormatHelper.formatOrganizationWithCvr);
+        } 
+
+        private loadData(): ng.IPromise<void> {
+            if (this.localAdmins.length > 0) {
+                this.resetLocalAdminData();
+            }
+            
+            return this.organizationRightService.getAllByRightsType(Services.Organization.AdminRightsType.LocalAdmin).then(response => {
+                this.localAdmins.pushArray(response.map((row, index) => {
+                    return {
+                        id: index,
+                        name: row.user.fullName,
+                        organization: row.organizationName,
+                        email: row.userEmail,
+                        objectContext: row
+                    } as ILocalAdminRow;
+                }));
+
+                this.emitLocalAdminRightsUpdatedEvent();
+            });
+        }
+
+        private resetLocalAdminData() {
+            const arrayLength = this.localAdmins.length;
+            this.localAdmins.splice(0, arrayLength);
+        }
+
+        private emitLocalAdminRightsUpdatedEvent() {
```

Comment:
the problem with this name is the overlap with the $emit function... rename to "publish...Event"